### PR TITLE
Edit security-overview.adoc

### PR DESCRIPTION
**Pull request by rolfedh**
References https://issues.redhat.com/browse/QDOCS-562

### CODE REVIEW

**Review by sberyozkin — APPROVED** [review]

File: docs/src/main/asciidoc/security-overview.adoc
Comment (sheilamjones) [inline docs/src/main/asciidoc/security-overview.adoc:22]: Not entirely sure about this change. To me, the update changes the meaning slightly and perhaps is a little ambiguous.. 
IMHO, the original states that you are mapping credentials to a `SecurityIdentity` instance that (already) has the listed attributes, whereas the updated version reads that you are mapping credentials to a `SecurityIdentity` instance along with the listed attributes.

**Review by sheilamjones — COMMENTED** [review]
Just had one very small comment for your consideration. Otherwise, all looks great to me.

File: docs/src/main/asciidoc/security-overview.adoc
Comment (MichalMaler) [inline docs/src/main/asciidoc/security-overview.adoc:89]:
```suggestion
For more information, see the xref:resteasy-reactive.adoc#secure-serialization[JSON serialization] section of the Quarkus "Writing REST services with RESTEasy Reactive" guide.
```

**Review by MichalMaler — APPROVED** [review]

**Comment by sberyozkin** [comment]
@rolfedh Proposed suggestions LGTM, can be merged once they are applied

File: docs/src/main/asciidoc/security-overview.adoc
Comment (rolfedh) [reply docs/src/main/asciidoc/security-overview.adoc:22]: You're right. Good point.

File: docs/src/main/asciidoc/security-overview.adoc
Comment (rolfedh) [reply docs/src/main/asciidoc/security-overview.adoc:89]: Updated. Thanks!

**Review by rolfedh — COMMENTED** [review]
Good feedback. Thank you.

**Comment by rolfedh** [comment]
Maintainer: This PR has been reviewed and approved by both the subject matter expert, @sberyozkin, and the peer reviewers, @MichalMaler and @sheilamjones. Therefore, it is ready to merge.

**Comment by MichalMaler** [comment]
Hello @jmartisk , can you merge it for us? :)

**Review by jmartisk — APPROVED** [review]

**Pull request merged**
